Library/Homebrew/requirements/minimum_macos_requirement.rb

```diff
@@ -11,6 +11,6 @@ def initialize(tags)
   satisfy(:build_env => false) { MacOS.version >= @version }
 
   def message
-    "OS X #{@version.pretty_name} or newer is required."
+    "#{MacOS.os_name} #{@version.pretty_name} or newer is required."
```

Comment:
This will print hideous things like “OS X 10.12” if triggered on 10.11, as `MacOS.os_name` would need to be context-aware and print something adjusted to `@version` in this case (not the static `MacOS.version`). I'm also missing the companion change in `maximum_macos_requirement.rb`. :wink: